lib/ansible/parsing/vault/__init__.py

```diff
@@ -659,8 +660,7 @@ def decrypt_and_get_vault_id(self, vaulttext, filename=None, obj=None):
                 msg += "%s is not a vault encrypted file" % to_native(filename)
             raise AnsibleError(msg)
 
-        b_vaulttext, dummy, cipher_name, vault_id = parse_vaulttext_envelope(b_vaulttext,
-                                                                             filename=filename)
+        b_vaulttext, dummy, cipher_name, vault_id = parse_vaulttext_envelope(b_vaulttext, filename=filename)
```

Comment:
Just formatting?

Reply:
i might have done some while trying to trace where things were